kernels/copy_slice_from.cl

```diff
@@ -0,0 +1,14 @@
+__constant sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void copy_slice_from_3d( 
+    IMAGE_src_TYPE  src,
+    IMAGE_dst_TYPE  dst,  
+    const int       cst
```

Comment:
What does "cst" mean? Can we call it "slice" or something more descriptive?

Reply:
`cst` stand for constant, we can use something else. `slice` is more descriptive, true. We could also use `index` which would also be valide for other similar kernels

Reply:
Ok, I like "index" 🌞

Reply:
:+1: